src/Lyn.Protocol/Bolt1/PongMessageService.cs

```diff
@@ -18,7 +18,7 @@ public PongMessageService(ILogger<PongMessageService> logger, IPingPongMessageRe
 
         public async Task ProcessMessageAsync(PeerMessage<PongMessage> request)
         {
-            _logger.LogDebug($"Processing pong from with length {request.Message.BytesLen.ToString()}");
+            _logger.LogDebug($"Processing pong from {request.NodeId} with length {request.Message.BytesLen.ToString()}");
```

Comment:
we should avoid calling `ToString` on logs this happens anyway under the hood and if debug is not enabled its waisted resources